engine/src/test/java/org/camunda/bpm/engine/test/concurrency/ConcurrentTelemetryConfigurationTest.java

```diff
@@ -52,6 +46,8 @@ protected void runTest() throws Throwable {
         || (transactionIsolationLevel != null && !transactionIsolationLevel.equals(Connection.TRANSACTION_READ_COMMITTED))) {
       // skip test method - if database is H2
     } else {
+      // clean up the db property
+      TestHelper.deleteTelemetryProperty(processEngineConfiguration);
```

Comment:
Not directly related to the topic of this ticket: Why does the test method sleep for 2 seconds?

Reply:
`thread2` to make progress before `thread1` continues and finishes, otherwise, the concurrent situation might not happen if `thread1` is faster. I guess this can be rewritten and improved (I didn't come up with how).

Reply:
Yana and I discussed this and understood that the Thread.sleep is necessary. The threads block on the pessismistic lock and in the concurrency test we have no chance to be notified of that waiting, so the best solution to is to choose a timeout that makes it quite likely that a thread reaches that waiting point.